easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,28 @@
+name = 'TensorFlow'
+version = '1.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-1.4.0_swig-env.patch',
+    'TensorFlow-%(version)s_fix-Bazel-version-check.patch',
+]
+checksums = [
+    '0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8',  # v1.5.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.4.0_swig-env.patch
+    '2809eb0c905961490459ab347ad78bad9a86f3c6c1c8bd461117e779ad0e4b21',  # TensorFlow-1.5.0_fix-Bazel-version-check.patch
```

Comment:
@boegel could you abbreviate the patch comment here to something like `...-1.5.0_` etc., to avoid excessive line length?